3p-ecosystem/src/main/java/com/google/homesampleapp/screens/settings/SettingsDeveloperUtilitiesFragment.kt

```diff
@@ -74,4 +107,114 @@ class SettingsDeveloperUtilitiesFragment :
       findNavController().popBackStack()
     }
   }
+
+  // -----------------------------------------------------------------------------------------------
+  // Composables
+
+  @Composable
+  private fun SettingsDeveloperUtilitiesScreen(
+  ) {
+    val showLogReposDialog = remember { mutableStateOf(false) }
```

Comment:
nit: use `by` to get rid of the `.value` ritual.

```kotlin
    var showLogReposDialog by remember { mutableStateOf(false) }
```

Reply:
Indeed. Thx.